### PR TITLE
[1.6] Tilegeneration uses --capabilities flag now

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

**Comment by yjacolin** [comment]
@sbrunner are you ok with my PR? :)

**Comment by sbrunner** [comment]
Why did it suddenly changed?

File: doc/integrator/tilegeneration.rst [hunk 1]
```diff
@@ -147,7 +147,7 @@ The main thing to do is to:
 
   .. prompt:: bash
 
-     .build/venv/bin/generate_controller --generate_wmts_capabilities
+     .build/venv/bin/generate_controller --capabilities
```
Comment (sbrunner) [inline doc/integrator/tilegeneration.rst]: => `--generate-wmts-capabilities`

**Comment by sbrunner** [comment]
Finally not, it's was just never used in 2 year ...

File: doc/integrator/tilegeneration.rst (hunk 1 above)
Comment (yjacolin) [reply doc/integrator/tilegeneration.rst]: both are ok indeed! but the one in doc is wrong.

**Comment by yjacolin** [comment]
close as it is correct in GMF 2.3 documentation.

**Pull request closed without merging**